pkg/cluster/install.go

```diff
@@ -220,28 +220,23 @@ func (m *manager) Update(ctx context.Context) error {
 			steps.Action(m.fixupClusterMsiTenantID),
 			steps.Action(m.ensureClusterMsiCertificate),
 			steps.Action(m.initializeClusterMsiClients),
-		)
-	}
-
-	s = append(s, steps.AuthorizationRetryingAction(m.fpAuthorizer, m.validateResources))
-
-	if m.doc.OpenShiftCluster.UsesWorkloadIdentity() {
-		s = append(s,
 			steps.AuthorizationRetryingAction(m.fpAuthorizer, m.clusterIdentityIDs),
 			steps.AuthorizationRetryingAction(m.fpAuthorizer, m.platformWorkloadIdentityIDs),
-			steps.Action(m.federateIdentityCredentials),
```

Comment:
Does this step just get skipped entirely on update now? Don't we need it somewhere?

Reply:
Good catch, I messed it up. Fixing it.

Reply:
Done